code/modules/mob/living/carbon/human/intoxication.dm

```diff
@@ -37,58 +21,72 @@ var/mob/living/carbon/human/alcohol_clumsy = 0
 		sleeping = 0
 		//Many of these parameters normally tick down in life code, but some parts of that code don't run in godmode, so this prevents a BST being stuck with blurred vision
 
-	if(intoxication > AE_DIZZY*SR) // Early warning
+	src.bac = round(intoxication/max(vessel.get_reagent_amount("blood"),1),0.01)
+
+	if(bac > INTOX_BUZZED*SR && bac < INTOX_MUSCLEIMP*SR && !(locate(/datum/modifier/buzzed) in modifiers))
+		src.show_message("<span class='notice'>You feel buzzed.</span>")
```

Comment:
You mean to use `to_chat(src, "message")`. `show_message` has unintended side-effects. Specifically, when unconscious, you get this message: `"<I>... You can almost hear someone talking ...</I>"`. Which is not befitting a status message.